lms/djangoapps/courseware/models.py

```diff
@@ -189,7 +194,6 @@ class XModuleStudentInfoField(XBlockFieldBase):
     """
     Stores data set in the Scope.preferences scope by an xmodule field
     """
-
```

Comment:
Nope. I was visually debugging through PyCharm and tried one of the call_stack_manager functionality, removed it after that. Should have done `git checkout` for this file. will take care of this.